tests/steamship_tests/endpoints/test_game_editor_endpoints.py

```diff
@@ -0,0 +1,43 @@
+from typing import Callable, Optional, Tuple
+
+import pytest
+import requests
+from steamship import Block, Steamship
+
+from api import AdventureGameService
+
+
+@pytest.mark.parametrize(
+    "invocable_handler_with_client", [AdventureGameService], indirect=True
+)
+def test_generate_preview(
+    invocable_handler_with_client: Tuple[
+        Callable[[str, str, Optional[dict]], dict], Steamship
+    ]
+):
+    invocable_handler, client = invocable_handler_with_client
+    task_dict = invocable_handler(
+        "POST", "generate_preview", {"field_name": "camp_image"}
+    )
+    block = Block.parse_obj(task_dict.get("data"))
+    url = f"{client.config.api_base}block/{block.id}/raw"
+    response = requests.get(url)
+    assert response.ok
+
+
+@pytest.mark.parametrize(
+    "invocable_handler_with_client", [AdventureGameService], indirect=True
+)
+def test_generate_suggestion(
+    invocable_handler_with_client: Tuple[
+        Callable[[str, str, Optional[dict]], dict], Steamship
+    ]
+):
+    invocable_handler, client = invocable_handler_with_client
+    task_dict = invocable_handler(
+        "POST", "generate_suggestion", {"field_name": "narrative_voice"}
+    )
+    block = Block.parse_obj(task_dict.get("data"))
```

Comment:
note: there is now a `blocks_from_invoke` in the SDK. not sure if that would help or not, as I forget the contract of `invocable_handler` at this point.